client/packages.go

```diff
@@ -32,15 +32,17 @@ import (
 
 // Snap holds the data for a snap as obtained from snapd.
 type Snap struct {
-	ID               string        `json:"id"`
-	Title            string        `json:"title,omitempty"`
-	Summary          string        `json:"summary"`
-	Description      string        `json:"description"`
-	DownloadSize     int64         `json:"download-size,omitempty"`
-	Icon             string        `json:"icon,omitempty"`
-	InstalledSize    int64         `json:"installed-size,omitempty"`
-	InstallDate      time.Time     `json:"install-date,omitempty"`
-	Name             string        `json:"name"`
+	ID            string    `json:"id"`
+	Title         string    `json:"title,omitempty"`
+	Summary       string    `json:"summary"`
+	Description   string    `json:"description"`
+	DownloadSize  int64     `json:"download-size,omitempty"`
+	Icon          string    `json:"icon,omitempty"`
+	InstalledSize int64     `json:"installed-size,omitempty"`
+	InstallDate   time.Time `json:"install-date,omitempty"`
+	Name          string    `json:"name"`
+	Publisher     string    `json:"publisher" help:"The publisher of the snap"`
```

Comment:
Hmm.
Could you add a
```go
type  Publisher struct {
 ID string `json:"id"`
 Username string `json:"username"`
 DisplayName string `json:"display-name,omitempty"`
}
```
and then have
```go
Publisher Publisher `json:"publisher"`
```
pretty please?

Reply:
Yes, this means some more work in daemon as well. But certainly doable.